de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/C_BPartner_Location_Label.sql

```diff
@@ -0,0 +1,41 @@
+DROP FUNCTION IF EXISTS report.C_BPartner_Location_Label( IN p_C_BPartner_Location_ID numeric, IN copies numeric );
+
+CREATE FUNCTION report.C_BPartner_Location_Label(p_C_BPartner_Location_ID numeric, IN copies numeric)
+  RETURNS TABLE
+  (
+    org_name    Character Varying,
+    bp_gln      Character Varying,
+    bp_gln_9_12 text,
+    addr_name   Character Varying,
+    address     text,
+    barcode     text
+  )
+AS
+$$
+SELECT t.*
+FROM generate_series(1, copies) i
+  CROSS JOIN LATERAL
+             (
+             SELECT
+               org.Name                                                                                as org_name,
+               COALESCE(NULLIF(bp_loc.gln, ''),
```

Comment:
```suggestion
               COALESCE(NULLIF(TRIM(bp_loc.gln), ''),
```
will also catch the case where a user entered some spaces but *think* they left the GLN empty

Reply:
@metas-ts Agree!